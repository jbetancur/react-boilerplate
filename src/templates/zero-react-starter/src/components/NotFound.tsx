import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
	display: flex;
	justify-content: center;
	align-items: center;
	padding: 80px;
	font-size: 92px;
	opacity: 0.4;
`;

const NotFound404: React.FC = () => (
	<Container role="heading" aria-label="heading">
		Page Not Found
	</Container>
);

export default NotFound404;
