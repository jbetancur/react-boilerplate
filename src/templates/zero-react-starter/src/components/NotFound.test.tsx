import React from 'react';
import { render } from '@testing-library/react';
import NotFound from './NotFound';

test('should render correctly', () => {
	const { container } = render(<NotFound />);

	expect(container.firstChild).toMatchSnapshot();
});
