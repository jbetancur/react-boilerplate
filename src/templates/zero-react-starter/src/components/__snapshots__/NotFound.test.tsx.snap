// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 80px;
  font-size: 92px;
  opacity: 0.4;
}

<div
  aria-label="heading"
  class="c0"
  role="heading"
>
  Page Not Found
</div>
`;
